test/e2e/secondary_network/network-attachment-definition-crd.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
The NetworkAttachmentDefinition CRD we use is trimmed version and we do not need the entire defs for our need. that was the reason I preferred to keep a copy in antrea instead of import as part of other dependent yamls.
All these will be pushed into the test execution cluster and gets deployed there.

Reply:
Sure, but I meant should we include whatever needed in the Antrea deployment YAML, instead of importing it in tests. Otherwise, how users are going to consume the feature. Do we expect them to import the CRD themselves?